Includes/xbe.h

```diff
@@ -0,0 +1,25 @@
+#ifndef XBE_H
+#define XBE_H
+
+#define XBETYPESTRING 0x48454258
+#define XBENAMESIZE 0x028
+#define SECTOR_SIZE 0x1000
+#define BASEADDRESS 0x10000
+
+typedef struct _XBE{
+  uint32_t type;
+  uint8_t  padding0[0xFF];
```

Comment:
nit: I personally name my padding / alignment structs by their offset (in structs like this). So this would be `padding_4` as it's at offset 4 (I typically use decimal offsets). This avoids name duplicates if you have to add more padding later (instead of having to call it `padding0b`)